### PR TITLE
LPD-3272 If versioning is enabled, use the new cpInstance externalRef…

### PR DESCRIPTION
**Pull request by brianikim**
…erenceCode

https://liferay.atlassian.net/browse/LPD-3272

@andrea-ale-sbarra

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ehi @brianikim can you please add some automation? 
Thank you!

**Comment by brianikim** [comment]
ci:test:sf

File: modules/test/playwright/pages/commerce/commerceCatalogSystemSettingsPage.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	readonly systemSettingsPage: SystemSettingsPage;
+	readonly updateButton: Locator;
+
+	constructor(page: Page) {
+		this.enabledButton = page.getByLabel('enabled');
+		this.systemSettingsPage = new SystemSettingsPage(page);
+		this.updateButton = page.getByRole('button', {name: 'Update'});
+	}
+
+	async configureProductVersioning() {
```
Comment (andrea-ale-sbarra) [inline modules/test/playwright/pages/commerce/commerceCatalogSystemSettingsPage.ts:22]: I would call this toggleProductVersioning rather then configureProductVersioning

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
@andrea-ale-sbarra , I am going to request assistance from @smottal on the playwright failure because it passes everytime locally.

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:stable

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by andrea-ale-sbarra** [comment]
this is failing https://testray.liferay.com/home/-/testray/case_results/2559563281?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2559561345%26testrayRunId%3D2559562903%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc can you please check?
Thank you!

**Comment by brianikim** [comment]
https://github.com/liferay-commerce/liferay-portal/pull/4856

**Pull request closed without merging**